packages/nextjs/src/config/webpack.ts

```diff
@@ -377,13 +382,27 @@ function checkWebpackPluginOverrides(
  *
  * @param entryPointName The name of the entry point in question
  * @param isServer Whether or not this function is being called in the context of a server build
+ * @param excludedServersideEntrypoints A list of excluded serverside entrypoints
  * @returns `true` if sentry code should be injected, and `false` otherwise
  */
-function shouldAddSentryToEntryPoint(entryPointName: string, isServer: boolean): boolean {
+function shouldAddSentryToEntryPoint(
+  entryPointName: string,
+  isServer: boolean,
+  excludedServersideEntrypoints: (string | RegExp)[],
```

Comment:
L: If we set a default value here, we don't have to make sure to include the `?? []` every time we call this function.

```suggestion
  excludedServersideEntrypoints: (string | RegExp)[] = [],
```